### PR TITLE
Add Key Binding setting shortcut

### LINKED ISSUE #884: Shortcuts / commands to edit keymaps
I'm coming from SublimeGit and shortcuts on status page is a bit different. I could not get used to it.

For example "d" is "diff" in SublimeGit, but in GitSavvy its "discard" shortcut. Everytime I mispress "d" key.

Is there any way to customize shortcuts?

### PR DESCRIPTION
**Pull request by stoivo**
Many Thanks to OdatNurd[1]
https://forum.sublimetext.com/t/open-key-sublime-keymap-with-edit-settings-with-specific-platform-in-mind/35604/3?u=stoivo

fix #884

[1] https://forum.sublimetext.com/u/OdatNurd

Two tabs on the left side
screen shot 2018-03-19 at 21 08 45

Should this be merged into `dev` or should to merge this one in to `release/2.17.0`?

### CODE REVIEW

**Comment by randy3k** [comment]
I have no objection to merge it to 2.17.0

**Comment by randy3k** [comment]
It seems that the command palette doesn't work, only the main menu works.

File: Default.sublime-commands [hunk 1]
```diff
@@ -6,6 +6,13 @@
             "base_file": "${packages}/GitSavvy/GitSavvy.sublime-settings",
             "default": "// Settings in here override those in \"GitSavvy/GitSavvy.sublime-settings\",\n\n{\n\t$0\n}\n"}
     },
+    {
+        "caption": "Preferences: GitSavvy Key Binding",
```
Comment (randy3k) [inline Default.sublime-commands]: Key Bindings

File: Main.sublime-menu [hunk 2]
```diff
@@ -17,6 +17,12 @@
                                     "base_file": "${packages}/GitSavvy/GitSavvy.sublime-settings",
                                     "default": "// Settings in here override those in \"GitSavvy/GitSavvy.sublime-settings\",\n\n{\n\t$0\n}\n"
                                 }
+                            }, {
+                                "caption": "Key Binding",
```
Comment (randy3k) [inline Main.sublime-menu]: Key Bindings

File: Default.sublime-commands (hunk 1 above)
Comment (stoivo) [reply Default.sublime-commands]: Agree

File: Main.sublime-menu (hunk 2 above)
Comment (stoivo) [reply Main.sublime-menu]: Agree

**Comment by randy3k** [comment]
It doesn't work with the dev channel as the command palette has been refactored.

**Comment by randy3k** [comment]
a891b30 should fix the issue, I have also filed a bug report https://github.com/SublimeTextIssues/Core/issues/2234

**Pull request merged**